Apps/phpassivetotal/passivetotal_consts.py

```diff
@@ -38,4 +39,10 @@
 
 PASSIVETOTAL_USING_BASE_URL = "Using url: {base_url}"
 
+# Rest API URL
 PASSIVETOTAL_REST_API_URL = "https://api.passivetotal.org/v2"
+
+# Consts for _get_error_message_from_exception
+PASSIVETOTAL_ERR_CODE_UNAVAILABLE = "Error code unavailable"
+PASSIVETOTAL_ERR_MSG_UNAVAILABLE = "Error message unavailable. Please check the asset configuration and|or the action parameters."
+PASSIVETOTAL_UNICODE_DAMMIT_TYPE_ERR_MESSAGE = "Error occurred while connecting to the PassiveTotal server. Please check the asset configuration and|or the action parameters."
```

Comment:
let's be consistent about using either `MSG` or `MESSAGE`

Reply:
Done